starboard/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ since the version previous to it.
 
 ## Version 16
 
+## Added new configuration constant `kHasPartialAudioFramesSupport`
```

Comment:
Can we also expose this to the H5VCC level? So that YTS can also verify the setting of this parameter.

Reply:
We can, but i'd do this in a follow-up PR. This change needs to land before Starboard 16 freeze, with h5vcc we have a bit more time.